### ISSUE TITLE
Determine whether or not we are using views_contextual_filters_or

### ISSUE THREAD

**Issue by wgilling** [asulibraries/islandora-repo#348]
This module was added in the early development of the collection stats and collection search, and was referenced in this pull request https://github.com/asulibraries/islandora-repo/pull/284 but we seem to no longer be using it at all.

**Comment by wgilling** [comment]
I uninstalled the module in my VM, cleared the cache, did a grep search of the config/sync folder and code directories, made sure that my objects would expose whether or not there was an issue with this module being removed and ....

```
config/sync$ grep -ir "contextual" .
./core.extension.yml:  contextual: 0
./core.extension.yml:  views_contextual_filters_or: 0
./views.view.paragraphs_to_update.yml:          contextual_filters_or: false
./views.view.solr_search_content.yml:          contextual_filters_or: 0
```

Do the last two results above have different values for the contextual_filters_or option? Both of these are underneath a query section but the paragraphs_to_update view has it under "query - type: views_query - options" but the solr_search view has it under the "query - type: search_api_query - options" section.

As long as we wanted the "items" count on a collections page (the first stats box) to include items that are related to the collection via "Primary member of" as well as "Additional memberships", while the search page itself that this links to has ONLY the items that are related via the "Primary member of" relationship only.

**Comment by wgilling** [comment]
Now, the latest `develop` code was already exhibiting the issue with "items" count on a collection page to not match the actual underlying search count since that view is only considering the "Primary member of" (ancestors).

What is strange is that even after the module is uninstalled, the cache is cleared and then configs are exported, the two views mentioned above (`views.view.solr_search_content.yml` and `views.view.paragraphs_to_update.yml`) the exported configurations files for them STILL REFERENCE "contextual_filters_or" within them and have the same values as they indicate above. I even tried removing via the UI instead, exported configs still had a setting like `contextual_filters_or: 0` ... finally, I removed the module completely by `composer remove drupal/views_contextual_filters_or` and the settings persist. Can I just put any value into a config and it will be there forever?

**Comment by elizoller** [comment]
closed with #376
